admin_tools/dashboard/templates/admin/app_index.html

```diff
@@ -1,19 +1,9 @@
-{% extends "admin/index.html" %} 
-{% load i18n admin_tools_dashboard_tags %} 
-
-{% if not is_popup %}
-
+{% extends "admin:admin/app_index.html" %} 
+{% load admin_tools_dashboard_tags %} 
+{% block extrastyle %}
```

Comment:
(sorry - this comment is actually more applicable to styles than it is dashboard)